frontend/src/components/AnalysisPanel/AnalysisPanel.tsx

```diff
@@ -17,11 +19,25 @@ interface Props {
     citationHeight: string;
     answer: AskResponse;
     fileType: string;
+    onHideTab: () => void;
 }
 
 const pivotItemDisabledStyle = { disabled: true, style: { color: "grey" } };
 
-export const AnalysisPanel = ({ answer, activeTab, activeCitation, citationHeight, className, onActiveTabChanged, fileType }: Props) => {
+const closeButtonStyle = {
```

Comment:
@PedroLabrador this style shouldn't be here :`frontend/src/components/AnalysisPanel/AnalysisPanel.module.css `instead?